test/integration_test.py

```diff
@@ -1035,3 +1043,78 @@ def test_local_filtered_index_endpoints(self):
                                              catalog=self.catalog)).url
         response = requests.get(url)
         self.assertEqual(200, response.status_code)
+
+
+class CanBundleScriptIntegrationTest(AzulTestCase):
+    scripts_root = f'{config.project_root}/scripts'
+
+    def _test_catalog(self, catalog: config.Catalog):
+        with self.subTest(catalog=catalog.name,
+                          repository=catalog.plugins['repository']):
+            fqid = self.bundle_fqid(catalog.name)
+            with tempfile.TemporaryDirectory() as d:
+                self._can_bundle(source=str(fqid.source.spec),
+                                 uuid=fqid.uuid,
+                                 version=fqid.version,
+                                 output_dir=d)
+                parts = ['manifest', 'metadata']
+                expected_files = sorted(f'{fqid.uuid}.{part}.json' for part in parts)
+                generated_files = sorted(os.listdir(d))
+                self.assertListEqual(generated_files, expected_files)
```

Comment:
This ensures we have files with the right name, but it might also be worth checking something about their structure. For example asserting that every file in the metadata.json is listed in the manifest. Up to you if you want to do this though.

Reply:
This was an excellent suggestion, I was overwhelmed by the idea of verifying the correctness of the entire bundle contents but this simple check found a bug in the staging area: https://github.com/DataBiosphere/hca-metadata-api/issues/52